src/main/scala/com.snowplowanalytics.sauna/apis/Sendgrid.scala

```diff
@@ -177,4 +192,382 @@ object Sendgrid {
       case Left(errors) => Left("Cannot extract custom fields information: " + errors.toString)
     }
   }
+
+  /**
+   * Represents an email recipient - may contain the recipient's name,
+   * must contain their email.
+   */
+  case class SendgridEmailObject(
+    email: String,
+    name: Option[String]
+  )
+
+  implicit val sendgridEmailObjectReads: Reads[SendgridEmailObject] = (
+    (JsPath \ "email").read[String] and
+      (JsPath \ "name").readNullable[String]
+    ) (SendgridEmailObject.apply _)
+  implicit val sendgridEmailObjectWrites: Writes[SendgridEmailObject] = Json.writes[SendgridEmailObject]
```

Comment:
Since there is no special logic, I thought json4s would take care of it by itself, I guess not :/ (circe ftw)